lib/defaults.rb

```diff
@@ -7,6 +7,7 @@ module Ffmprb
   Filter.silence_noise_max_db = -40
 
   Process.duck_audio_silent_min = 3
+  Process.duck_audio_audible_sound_min = 1
```

Comment:
I'm sticking to `duck_audio_transition_length` default value here as it seems a bit weird to make a fade in/out which is longer than the sound in between, correct me if I'm wrong..